### PR TITLE
Feat/eae 36 my deltares be

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: src/views/Account.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      // Go to login endpoint in BE. Use the route 'callback' to handle the
+      // authorization on redirect and set session cookies
+      return `${process.env.VUE_APP_EDITOR_SERVER}/login?redirect_uri=${window.location.origin}/callback`
+    },
+    logoutUrl () {
+      // Remove session cookies and use logout endpoint from BE
+      return `${process.env.VUE_APP_EDITOR_SERVER}/logout`
+    },
+    name () {
+      return _.get(this.user, 'name') || '--'
```
Comment (SiggyF) [inline src/views/Account.vue:53]:
```suggestion
      return _.get(this.user, 'name', '--')
```

File: src/views/auth/Callback.vue [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<template>
+  <div />
+</template>
+
+<script>
+export default {
+  mounted () {
+    const authUrl = `${process.env.VUE_APP_EDITOR_SERVER}/auth${window.location.search}`
+    window.location.assign(authUrl)
```
Comment (SiggyF) [inline src/views/auth/Callback.vue:9]: is this normal?

**Review by SiggyF — APPROVED** [review]
minor suggestion/comment

File: src/views/Account.vue (hunk 1 above)
Comment (CindyvdVries) [reply src/views/Account.vue:53]: cool, didn't know that was an option :D

**Pull request merged**